adoc/bp_chap_expanded_support.adoc

```diff
@@ -430,72 +428,97 @@ Some actions are not yet supported:
+
+. Ensure that you have the appropriate software channels available on your system.
+In the {productname} {webui}, navigate to menu:Software[Channel List  > All].
+You should see a base channel and a child channel for your architecture, for example:
 +
 ----
-sudo visudo
+ ubuntu-18.04-pool for amd64
+ |
+ +- Ubuntu-18.04-SUSE-Manager-Tools for amd64
 ----
+. Create custom repositories to mirror the {ubuntu} packages.
+For more information on creating custom repositories, see xref:modules/administration/pages/channel-management.adoc[].
+For example:
 +
-Grant [command]``sudo`` access to the user by adding this line to the [filename]``sudoers`` file. Replace [systemitem]``<user>`` with the name of the user that will bootstrap the client in the {webui}:
+For `main`:
+
+* Repository Label: ubuntu-bionic-main
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/
+* Repository Type: deb
 +
-----
-<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
-----
+For `main-updates`:
 
-.Procedure: Preparing an {Ubuntu} 18.04 Client for Registration
+* Repository Label: ubuntu-bionic-main-updates
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/
+* Repository Type: deb
 
-. On the client, open the [filename]``/etc/apt/sources.list.d/suma_client_tools.list`` file, and add this line:
+. Create custom channels under the `pool` channel, mirroring the vendor channels.
+For more information on creating custom channels, see xref:modules/administration/pages/channel-management.adoc[].
+Ensure the custom channels you create have `AMD64 Debian` architecture.
++
+For example:
 +
 ----
-deb http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/3.2:/Ubuntu18.04-SUSE-Manager-Tools/xUbuntu_18.04/ /
+ ubuntu-18.04-pool for amd64 (vendor channel)
+ |
+ +- Ubuntu-18.04-SUSE-Manager-Tools for amd64 (vendor channel)
+ |
+ +- ubuntu-18.04-amd64-main (custom channel)
+ |
+ +- ubuntu-18.04-amd64-main-updates (custom channel)
 ----
-+
-. Edit the [filename]``sudoers`` file:
-+
+. Associate the custom channels with the appropriate custom repositories.
+. Synchronize the new custom channels.
+You can check the progress of your synchronization from the command line with this command:
 ----
-sudo visudo
+tail -f /var/log/rhn/reposync.log /var/log/rhn/reposync/*
 ----
-+
-Grant [command]``sudo`` access to the user by adding this line to the [filename]``sudoers`` file. Replace [systemitem]``<user>`` with the name of the user that will be used to bootsrap the client in the {webui}:
+. To use bootstrap with {ubuntu}, you will need to create a bootstrap repository.
+You can  do this from the command line with [command]``mgr-create``:
 +
 ----
-<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
+mgr-create-bootstrap-repo --with-custom-channels
 ----
 
 
 
-.Procedure: Synchronizing {ubuntu} Repositories
-
+.Procedure: Granting Root User Access
```

Comment:
I think this step needs to be done on the client. It is unclear that it needs to happen there.
All other steps above are for the server.

@mateiw is this correct?